devstack/settings

```diff
@@ -37,11 +37,23 @@ enable_service group-policy
 disable_service tempest
 ENABLED_SERVICES+=,heat,h-api,h-api-cfn,h-api-cw,h-eng
 # NFP services
-enable_service orchestrator,proxy,proxy_agent,base_configurator
+enable_service orchestrator,proxy,proxy_agent,config_orch
 
 # Deployment preferences
 SYSLOG=${SYSLOG:-True}
 
 # Skip exercises by default (can be overridden in local.conf)
 SKIP_EXERCISES=${SKIP_EXERCISES:-volumes,trove,swift,sahara,euca,bundle,boot_from_volume,aggregates,zaqar,client-env,neutron-adv-test,floating_ips,client-args,horizon,sec_groups,gbp_servicechain,gbp_heat}
 
+# Service VM names
+ConfiguratorQcow2Image=/home/stack/service_images/configurator-mitaka-master.qcow2
+VyosQcow2Image=/home/stack/service_images/vyos.qcow2
+HaproxyQcow2Image=/home/stack/service_images/haproxy.qcow2
+
+EXT_NET_NAME=ext-net
+EXT_NET_SUBNET_NAME=ext-net-subnet
+EXT_NET_GATEWAY=192.168.102.254
+EXT_NET_ALLOCATION_POOL_START=192.168.102.81
+EXT_NET_ALLOCATION_POOL_END=192.168.102.90
+EXT_NET_CIDR=192.168.102.0
+EXT_NET_MASK=24
```

Comment:
+EXT_NET_GATEWAY=192.168.102.254
+EXT_NET_ALLOCATION_POOL_START=192.168.102.81
+EXT_NET_ALLOCATION_POOL_END=192.168.102.90
+EXT_NET_CIDR=192.168.102.0
+EXT_NET_MASK=24

The above should be in local.conf and should be mandatory input.

+EXT_NET_NAME=ext-net
+EXT_NET_SUBNET_NAME=ext-net-subnet

The above names can be given in local.conf also. So, in setting read like this
+EXT_NET_NAME=${EXT_NET_NAME:-ext-net}
+EXT_NET_SUBNET_NAME=${EXT_NET_SUBNET_NAME:-ext-net-subnet}